integration/e2e/features/table-of-contents.feature

```diff
@@ -0,0 +1,16 @@
+@Toc
+@android @iphone @desktop
+Feature: Displaying Table of Contents
+  In order to easily navigate in a lengthy publication
+  As a user
+  I want a table of contents
+
+  Background:
+    Given the viewer is opened with a publication which include a table of contents
+
+  Scenario: Display Table of Contents
```

Comment:
Use Given, When, Then

Testing style bold is not necessary

The feature should test the user selecting a item in the TOC